### PR TITLE
Workshop On Hosting, Custom Domains and More 

### PR DESCRIPTION
**Pull request by aaryanporwal**
Resubmission of : #1560

### CODE REVIEW

File: workshops/hosting-workshop/README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+### THE END :)
+
+In this workshop we skimmed through a lot of topics, so take your own time, and understand stuff better. There are a few points I'd like to mention though:
+
+In the workshop we put our website's code in a repository on GitHub that was public, so essentially anyone can copy the website, and that can be harmful, so there are two solutions for this:
+
+One is that if you want to make your website Open Source (that is want the source code to be open to public), you can choose amongst one of the Open Source Licenses. To help with this, GitHub and the community made this awesome website that you can check out [here](https://choosealicense.com).
+
+Other is that if you want to make your website Closed Source (that is don't want the source code to be open to public), you can simply make your GitHub repository private.
```
Comment (sampoder) [inline workshops/hosting-workshop/README.md:160]:
```suggestion
The other solution is to make your website closed source (if you don't want the source code to be public). To achieve this you can simply make your GitHub repository private.
```

**Review by sampoder — CHANGES_REQUESTED** [review]
Hey Aaryan!

Thank you for submission, alongside my comments here are a couple of pointers.

- The grammar needs significant improvement. Read the workshop out slowly to yourself and fix up any issues that block the flow.
- This workshop probably needs to actually introduce the key terms in it's title and what they mean.
- This workshop needs to be more flexible to accommodate for the different situations clubs are in.

Thanks,

Sam 🚀

File: workshops/hosting-workshop/README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+# Deploying, Hosting, Continuous Deployment and Custom Domains
+
+In this workshop we'll be learning how to push your website on GitHub (A website that provides hosting for software development and version control using Git), hosting it on [Netlify](https://www.netlify.com) with a custom domain from [namecheap](https://www.namecheap.com).
+
+## Part 1: The Prerequisites
+
+1. A GitHub Account (it's just as easy as signing up for any other website!)
+
+2. Next, we need [GitHub's student developer pack](https://education.github.com/pack) for this Workshop so that we can take advantage of namecheap's free 1 year custom domain name, for this you can take advantage of Hack Club's [Hack Pack](https://toolbox.hackclub.com/hack_pack) to instantly get GitHub's student developer pack!
```
[the remark this answers is not in the document]
Comment (aaryanporwal) [reply workshops/hosting-workshop/README.md:16]: @sampoder When I wrote the workshop, it was up and running :(
Do you think, I should just remove it altogether?

File: workshops/hosting-workshop/README.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+4. Then, click on "Deploy Site", this will deploy your website on Netlify's CDN ![click deploy site](https://cloud-1lhe4c42f.vercel.app/0image.png)
+5. Next, click on "Domain Settings" to add a custom domain to your deployment ![click domain settings](https://cloud-pdrxlnmln.vercel.app/0image.png)
+6. In the Domain Settings, you can see a strange domain provided by Netlify, you can click on that to see how your project looks like!
+
+In the next part we'll add a custom domain to our app!
+
+## Part 5: Custom Domain
+
+Well, we did see a domain for our website on Netlify, but well it sucks, seriously, if I asked you to close your eyes and repeat back the domain address, you probably couldn’t. But don't worry Custom Domains are our saviours here!
+First, get a free one year custom domain from [Namecheap Education](https://nc.me) using your GitHub account.
```
[the remark this answers is not in the document]
Comment (aaryanporwal) [reply workshops/hosting-workshop/README.md:117]: Yeah, that's the beauty of SDP, you don't need credit card or anything for buying a domain from Namecheap!

[the remark this answers is not in the document]
Comment (aaryanporwal) [reply workshops/hosting-workshop/README.md:117]: You can get a `.me` domain free for one year on namecheap!

File: workshops/hosting-workshop/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sampoder) [reply workshops/hosting-workshop/README.md:16]: for now probably :/

**Pull request closed without merging**